zhaquirks/sonoff/snzb03p.py

```diff
@@ -0,0 +1,64 @@
+"""Sonoff SNZB-03P - Zigbee motion sensor."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import EntityPlatform, EntityType, NumberDeviceClass, QuirkBuilder
+from zigpy.quirks.v2.homeassistant import UnitOfTime
+import zigpy.types as t
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class LastIlluminationState(t.enum8):
+    """Last measureed state of illumination enum."""
+
+    Dark = 0x00
+    Light = 0x01
+
+
+class SonoffIlluminationCluster(CustomCluster):
+    """Custom Sonoff illumination cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        last_illumination_state = ZCLAttributeDef(
+            id=0x2001,
+            type=LastIlluminationState,
+            access="rp",
+            is_manufacturer_specific=True,
+        )
+
+    @property
+    def _is_manuf_specific(self):
+        return False
```

Comment:
Z2M seems to specify a manufacturer code: https://github.com/Koenkk/zigbee-herdsman-converters/blob/216ba1a56f1b21fda7de7204839f0fb838df0b85/src/devices/sonoff.ts#L1162
Do we even need this at all? By default, zigpy will use the manufacturer code for the device if the attribute is marked as `is_manufacturer_specific` (it is) and if the cluster is in the manufacturer specific range (it is).

I guess this is only really needed if we explicitly poll/read the attribute ourselves though (through the "Manage Zigbee device -> Clusters" debug menu).